server/components/form-settings/form-settings.html

```diff
@@ -24,7 +24,7 @@
 
 {% call fieldset({ legend: 'Main personal information', sronly: true }) %}
 
-  {{ input({
```

Comment:
something went wrong with the code formatting. There are changes with the indenting

Reply:
Fixed it, could you check if it is good now?